File: backend/src/main/java/org/jdk/project/dto/config/ImageUpsertRequest.java
package org.jdk.project.dto.config;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class ImageUpsertRequest {
  private Long loungeId;
  private String imgName;
  private String imgType;
  private String img;
  private Integer width;
  private Integer height;
  private Integer enable;
  private String remark;
}

File: backend/src/main/kotlin/org/jdk/project/dto/config/ImageUpsertRequest.kt
package org.jdk.project.dto.config

data class ImageUpsertRequest(
    var loungeId: Long? = null,
    var imgName: String? = null,
    var imgType: String? = null,
    var img: String? = null,
    var width: Int? = null,
    var height: Int? = null,
    var enable: Int? = null,
    var remark: String? = null,
)
